index.js

```diff
@@ -34,11 +34,21 @@ import ethUtil from 'ethereumjs-util';
 import Transaction from './quarkchain-ethereum-tx';
 
 let Web3;
+/* eslint-disable */
 if (typeof window === 'undefined') {
-  Web3 = require("web3"); // eslint-disable-line
+  Web3 = require('web3');
```

Comment:
condition from line 40 to 44 can be combined

Reply:
done~

Reply:
the condition of line 38 and line 40 can also be combined, right?

Reply:
yep......